src/main/java/lotto/domain/matchkind/WinningKind.java

```diff
@@ -2,7 +2,7 @@
 
 import java.util.Arrays;
 
-public enum LottoMatchKind {
+public enum WinningKind {
     LOWER_THAN_THREE(0, false, 0),
     THREE(3, false, 5000),
     FOUR(4, false, 50000),
```

Comment:
언더바를 활용하면 가독성을 향상시킬 수 있을 것 같아요 :)
```suggestion
    LOWER_THAN_THREE(0, false, 0),
    THREE(3, false, 5_000),
    FOUR(4, false, 50_000),
```